tests.py

```diff
@@ -282,6 +294,29 @@ def test_main_to_stdout_no_debug(self):
         logger = logging.getLogger("anonip")
         self.assertEqual(logger.level, 30)
 
+    def test_main_reading_from_input_file(self):  # TODO use tmp_path
+        input_filename = "/tmp/anonig-input.txt"  # FIXME
```

Comment:
Would be nice to have this fixed before merging.

Reply:
Thanks for spotting this. I fixed this, see https://github.com/DigitaleGesellschaft/Anonip/pull/17#issuecomment-547524940